File: native-ui/app/src/main/java/com/example/cityinyourpocket/model/Event.kt
package com.example.cityinyourpocket.model

import java.io.Serializable

class Event(var name: String,
            var description: String,
            var hostUserName: String,
            var location: EventLocation,
            var startDate: String,
            var endDate: String,
            var openHour: String,
            var evId: Long): Serializable{

}

File: non-native-ui/model/event.js


export class Event{
    constructor(name, description, hostUserName, location, startDate, endDate, openHour, id){
        this.name = name
        this.description = description
        this.hostUserName = hostUserName
        this.location = location
        this.startDate = startDate
        this.endDate = endDate
        this.openHour = openHour
        this.id = id
    }
}